library/Haste/Dca/AjaxOperations.php

```diff
@@ -303,4 +305,16 @@ private function getOptions(array $hasteAjaxOperationSettings)
     {
         return (array) $hasteAjaxOperationSettings['options'];
     }
+
+    private function getVersionEditUrl(int $id, string $operation)
```

Comment:
Haste can be still used by PHP 5.6 systems so please don't use strict types here.

Reply:
It's really difficult not to use this new stuff anymore